lib/discourse_event.rb

```diff
@@ -21,6 +21,16 @@ def self.on(event_name, &block)
         raise_error: true,
       )
     end
+
+    if event_name == :user_badge_removed
```

Comment:
`elsif` is slightly shorter?

```suggestion
    elsif event_name == :user_badge_removed
```

Reply:
I didn't want to add it to the existing block because the `site_setting_saved ` deprecation notice probably needs to be removed. I wanted to simply yank it out in a follow up PR after confirming.